test/framework/modules.py

```diff
@@ -514,7 +514,7 @@ def test_modules_tool_stateless(self):
 
         if isinstance(self.modtool, Lmod):
             # GCC/4.6.3 is nowhere to be found (in $MODULEPATH)
-            load_err_msg = r"The following module\(s\) are unknown"
+            load_err_msg = r"The[\s\n]*following[\s\n]*module\(s\)[\s\n]*are[\s\n]*unknown"
```

Comment:
Why? Lmod is wrapping output lines differently in a headless (0-width) terminal, so the whole message doesn't fit on a single line anymore, it's wrapped.